source/commands.js

```diff
@@ -37,12 +37,17 @@ async function applyCommand (name, env, options) {
 const list = {
   create: {
     aliases: ['new'],
-    meta: '[<directory>]',
+    meta: '[<directory>] [yes|y]',
```

Comment:
Since this is a flag, we do not include it in the task description. You can get it on `--help`.

Reply:
sorry for this mistakes
thanks for your help

updated commit